Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -81,9 +81,12 @@ async function run() {
     var connectionType = tl.getInput("connectionType", true);
     var isKubConfigRequired = isKubConfigSetupRequired(command);
     var kubectlCli: kubernetescli;
-    var externalAuth = connectionType === "None" && tl.getVariable("KUBECONFIG");
+    var externalAuth = connectionType === "None" && (command === "install" || command === "upgrade");
+    if (externalAuth && !tl.getVariable("KUBECONFIG")) {
+        tl.error("KUBECONFIG kube configuration file path must be set when command is install or upgrade.");
```

Comment:
missing when "Connection Type" is none

"...when Connection Type is none and command is install or upgrade"

Reply:
updated